src/Receiver.cc

```diff
@@ -436,7 +467,7 @@ Receiver::Message::strip(size_t count)
 void
 Receiver::Message::release()
 {
```

Comment:
This method (and the called destructor), races with `handleDataPacket()` because of the reduced bucket lock scope in both here and `handleDataPacket()`; this can result in `handleDataPacket()` operating on a destructed Message.

Reply:
should be fixed now